staff/joseluis-juste/JS/proyect2.js

```diff
@@ -0,0 +1,238 @@
+Array.prototype.showAll = function()
+	{
+		this.forEach(function(x){
+
+			console.log("ID: " + x.id + ", Origen: " + x.from + ", Destino: " + x.to + ", Coste: " + x.cost + ", Escala: " + x.scale);
+		});
+	};
+
+	var flights = 
+	[
+		{id: 00, to: "New York", from: "Barcelona", cost: 700,scale: false},
+		{id: 01, to: "Los Angeles", from: "Madrid", cost: 1100,scale: true},
+		{id: 02, to: "Paris", from: "Barcelona", cost: 210,scale: false},
+		{id: 03, to: "Roma", from: "Barcelona", cost: 150,scale: false},
+		{id: 04, to: "London", from: "Madrid", cost: 200,scale: false},
+		{id: 05, to: "Madrid", from: "Barcelona", cost: 90,scale: false},
+		{id: 06, to: "Tokyo", from: "Madrid", cost: 1500,scale: true},
+		{id: 07, to: "Shangai", from: "Barcelona", cost: 800,scale: true},
+		{id: 08, to: "Sydney", from: "Barcelona", cost: 150,scale: true},
+		{id: 09, to: "Tel-Aviv", from: "Madrid", cost: 150,scale: false}
+	];
+
+	function Admin(conf)
+	{
+		do
+		{
+			conf.option_admin = prompt("Desea anadir o eliminar un vuelo: [anadir | eliminar]");
+			if ((conf.option_admin !== "anadir") && (conf.option_admin !== "eliminar") && (conf.option_admin !== null))
+			{
+				console.log("");
+				console.log("El valor introducido es incorrecto");
+				continue;
+			}
+			if (conf.option_admin === null)
+			{
+				continue;
+			}
+			switch(conf.option_admin)
+			{
+				case "anadir":
+
+					do
+					{
+						conf.flight.from = prompt("Introduzca el origen");
+						conf.flight.to = prompt("Introduzca el destino");
+						conf.flight.cost = prompt("Introduzca el coste");
+						if (isNaN(conf.flight.cost))
+						{
+							console.log("");
+							console.log("El coste introducido no es un número");
+							conf.flight = {};
+							continue;
+						}
+						conf.flight.scale = confirm("¿Sera un vuelo con escala?");
+						conf.flight.id = flights.length === 0 ? 0 : flights[flights.length - 1].id + 1;
+						flights.push(conf.flight);
+						conf.flight = {};
+						console.log("");
+						flights.showAll();
+						console.log("");
+						console.log("");
+
+					}while(confirm("¿Desea anadir otro vuelo?"));
+
+				break;
+
+				case "eliminar":
```

Comment:
As a suggestion I would recommend you to use english instead of spanish !

Reply:
Fixed in commit JS SyntaxFixes.